src/components/job-overview/steps-list.js

```diff
@@ -40,43 +24,31 @@ const getStepIcon = (step) => {
     return track_changes;
   } else if (step.name.match(/^scan-(.+)$/)) {
     return record;
-  } else {
-    return radio_button_unselected;
   }
+
+  return radio_button_unselected;
 };
 
 export const StepsList = ({ appName, jobName, steps }) => {
   const namedSteps = steps ? steps.filter((s) => s.name) : [];
   return (
     <>
-      {namedSteps && (
-        <>
-          {namedSteps.length > 0 ? (
-            <React.Fragment>
-              <h4>Steps</h4>
-              <div className="steps-list__content">
-                {namedSteps.map((step) => (
-                  <div key={step.name} className="steps-list__step">
-                    <div className="steps-list__icon">
-                      <Icon data={getStepIcon(step)} />
-                      <span className="steps-list__divider-line"></span>
-                    </div>
-                    <StepSummary
-                      appName={appName}
-                      jobName={jobName}
-                      step={step}
-                    />
-                  </div>
-                ))}
+      <Typography variant="h4">Steps</Typography>
+      <div className="steps-list__content">
```

Comment:
use grid gap instead of margin
```suggestion
      <div className="grid grid--gap-medium">
```

Reply:
removed